ee/clickhouse/views/test/test_experiment_holdouts.py

```diff
@@ -52,7 +52,7 @@ def test_create_update_experiment_holdouts(self) -> None:
                     ],
                     "properties": [],
                 },
-                "holdout": holdout_id,
+                "holdout_id": holdout_id,
             },
```

Comment:
need to add the full holdout here too?

Reply:
nope, not on creation / editing experiments. Just the pk is enough, and we disallow creating a holdout within an experiment